app/scripts/directives/overview/virtualMachineRow.js

```diff
@@ -24,6 +24,76 @@
     templateUrl: 'views/overview/_virtual-machine-row.html'
   });
 
+  var RDP_PORT = 3389;
+
+  function fileDownload(data, fileName, mimeType) {
+    fileName = fileName || "vm.rdp";
+    mimeType = mimeType || "application/rdp";
+
+    var a = document.createElement('a');
+    if (navigator.msSaveBlob) { // IE10
+      return navigator.msSaveBlob(new Blob([data], { type: mimeType }), fileName);
+    } else if ('download' in a) { // html5 A[download]
+      a.href = 'data:' + mimeType + ',' + encodeURIComponent(data);
+      a.setAttribute('download', fileName);
+      document.body.appendChild(a);
+      a.click();
+      document.body.removeChild(a);
+      return true;
+    } else { // do iframe dataURL download (old ch+FF):
```

Comment:
I don't think we need to fallback for old versions here. We already depend on HTML5 file APIs elsewhere.

Reply:
removed